a10_octavia/controller/worker/tasks/handler_l7rule.py

```diff
@@ -26,55 +26,65 @@
 class L7RuleParent(object):
 
     def set(self, l7rule, listeners, vthunder):
-        """ Execute create l7rule """
+        """Execute create l7rule """
```

Comment:
Remove the space at the end

Reply:
Removing this line, as set is not expected to have this line, it should be only in CreateL7RuleTask